### PR TITLE
Polish Iris DNN tutorial.

### PR DESCRIPTION
**Pull request by shendiaomo**
1. `iris' -> 'Iris' except in code snippet
2. Number of hidden units of the 2nd layer in code snippet and  the description above is consistent now.
3. Add a wikipedia link to the Iris flow dataset.
4. The 'How to' list is changed to bulleted to be consistent with other tutorials.

### CODE REVIEW

**Review by weiguoz — DISMISSED** [review]
LGTM

File: doc/tutorial/iris-dnn.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Classify Iris Dataset Using DNNClassifer
 
 This tutorial demonstrates how to
-1. train a DNNClassifer on iris dataset.
-1. use trained DNNClassifer to predict iris class.
+- Train a DNNClassifer on the [Iris flower dataset](https://en.wikipedia.org/wiki/Iris_flower_data_set).
+- Use the trained DNNClassifer to predict the 3 species of Iris(Iris setosa, Iris virginica and Iris versicolor).
```
Comment (typhoonzero) [inline doc/tutorial/iris-dnn.md]: 3 -> three

File: doc/tutorial/iris-dnn.md [hunk 2]
```diff
@@ -26,13 +26,13 @@ limit 5;
 
 ## Train
 
-Let's train a DNNClassifier, which has two hidden layers where each layer has ten hidden units. This can be done by specifying the training clause for SQLFlow's extended syntax.
+Let's train a 3-class DNNClassifier, which has 2 hidden layers with 10 hidden units each. This can be done by specifying the training clause for SQLFlow's extended syntax.
```
Comment (typhoonzero) [inline doc/tutorial/iris-dnn.md]: Use English than numbers in formal documents should be better.

File: doc/tutorial/iris-dnn.md (hunk 1 above)
Comment (shendiaomo) [reply doc/tutorial/iris-dnn.md]: Done.

File: doc/tutorial/iris-dnn.md (hunk 2 above)
Comment (shendiaomo) [reply doc/tutorial/iris-dnn.md]: Done.

**Review by typhoonzero — APPROVED** [review]

**Pull request merged**